contents/blog/product-market-fit-game.md

```diff
@@ -0,0 +1,351 @@
+
+### Level 4 – Validate your product by retaining users
+
+Hooray, so people are showing up. Now you need to see if they're coming back or not. Figure out what your expected frequency is for usage, and check if that's roughly how often users are using your product. Users will return if you can repeatedly solve a problem for them.
+
+Avoid relying on things like reminder emails, or notifications purely designed to get users to log back into your product. If it takes this sort of effort to retain users, your product probablty isn't valuable to them. Focus _exclusively_ on building features that make your product more powerful, or easier to use.
+
+Once you've got some usage, get qualitative and quantiative feedback. Measure which features are being used the most, ask for feedback calls, and watch users using your product. There are plenty of good session replay tools... ours is [one of them](/session-replay)!
+
+If they're willing, ask them to perform tasks live in front of you, and ask them why they clicked certain things. You'll be shocked at how many ways people get confused – that's a _great_ problem to have if they're coming back.
+
+Start "closing the loop" with users. This is a simple process to improve your product:
+
+![product improvement loop](../images/blog/pmf-game-guide/product-loop.png)
+
+Closing the loop above is the way to cement your product-market fit because:
+
+1. **Your product will get better fast.** You should weight user feedback very heavily compared to your instincts around what to build. 
+
+2. **It will generate word of mouth growth.** The one thing you can compete on is speed – you can outperform any competitors by providing a ridiculously excellent experience to your early users, so they will tell their friends. 
+
+3. **Listening to users compounds.** Like a snowball rolling down a hill, the more you listen to users and act on their feedback, the more useful feedback they'll share with you. 
+
+#### Failure modes
+
+* **Users aren't activating.** This means although they _signed up_, but can't _configure_ / get started with your product, so they _never_ get any value from it. You have two options here:
+
+  1. Activate your users manually. The classic example is the founders of Stripe who [used to visit their first users in person](http://paulgraham.com/ds.html#Manual) to integrate their product for them. Do things that don't scale. The bonus of this approach is that you'll get deeper customer interaction, so you'll learn more and you'll get a better relationship which will help you get more feedback.
+  
+  2. Make your activation/onboarding simpler. Can you make it 10x simpler?
+
+* **You are getting conflicting feedback on what to build.** Go to Level 5, this will help clarify which users are your Ideal Customer Profile, and which aren't. Hopefully this exercise will help you pick a clearer direction.
```

Comment:
```suggestion
* **You are getting conflicting feedback on what to build.** Go to level 5, this will help clarify which users are your Ideal Customer Profile, and which aren't. This provides a clearer direction.
```